applets/applet_tables.rs

```diff
@@ -1,1048 +1,369 @@
+#![allow(unused_imports)]
```

Comment:
I think an easier way to do this without requiring `#![allow(unused_imports)]` would be to skip `use` statements altogether and to just write out the full function paths in the structs below. 

We can save that for a future PR though. This is already way, way better than before.